paper/paper.Rmd

```diff
@@ -23,15 +23,24 @@ knitr::opts_chunk$set(echo = TRUE)
+The default partitioner uses a correlation-based distance matrix to find the pair of features with the smallest distance between them; intraclass correlation (ICC) to measure information explained by the reduced variable; and scaled row means to reduce variables with a sufficient minimum ICC. In simulations, this ICC-based partitioner outperformed K-means and other approaches both in terms of speed and number of discoveries [@Millstein2019]. The ICC approach is fast and reliable, but partition also implements different strategies for directing, measuring, and reducing. For instance, the user can use principal components analysis to reduce variables instead of scaled row means. As the framework is agnostic to how partitions are being directed, measured, or reduced, custom partitioners are easy to implement. 
 
-While many tools exist in R for data reduction, including many functions in base R, partition offers a fast and flexible tool that addresses the need for interpretability and information retention in reduced variables. To our knowledge, it is the first package of its kind. Wide data, such as high-resolution genetic data, can be quickly reduced without sacrificing information. Additionally, as each feature of the data maps to a single cluster, it is easier to make inferences from the reduced data set. The flexibility of partition makes it adaptable to the needs of many data reduction strategies.
+Many tools exist in R for data reduction, including:
+
+* In the base R stats package, `princomp()` and `prcomp()` for PCA, `kmeans()` for K-Means Clustering, and `hclust()` for Hierarchical Clustering, among others [@r-base]
+* cluster, an R package for grouping data using Hierarchical Clustering and other strategies [@r-cluster]
+* Rtsne and tsne, R packages for t-SNE analysis [@r-Rtsne; @r-tsne]
+* uwot, an R package for Uniform Manifold Approximation and Projection (UMAP) dimensionality reduction [@r-uwot]
+
+partition differs from these tools mainly in that it constrains information loss; the way that variables map to reduced data may be easier to interpret than other data reduction approaches, as well. Notably, because the Partition framework is agnostic to the components of Direct-Measure-Reduce, users may apply tools from base R or other R packages at any of the three stages of the partition. 
```

Comment:
Suggestions:
partition differs from these tools mainly in that it constrains information loss; the way features map to reduced data is more interpretable than many other data reduction approaches. Notably, because in the Partition framework the components of Direct-Measure-Reduce are easily interchangeable, users may apply tools from base R or other R packages at any of the three stages of the partition.